aten/src/ATen/Parallel.h

```diff
@@ -97,3 +97,7 @@ CAFFE2_API void launch(const std::function<void()>& func);
 #if AT_PARALLEL_OPENMP
```

Comment:
Are `AT_PARALLEL_OPENMP` and `AT_PARALLEL_NATIVE` required to be mutually exclusive?

Reply:
Right, they are mutually exclusive